akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerSpec.scala

```diff
@@ -1061,7 +1061,7 @@ class HttpServerSpec extends AkkaSpec(
       "have a programmatically set timeout handler" in assertAllStagesStopped(new RequestTimeoutTestSetup(400.millis) {
         send("GET / HTTP/1.1\r\nHost: example.com\r\n\r\n")
         val timeoutResponse = HttpResponse(StatusCodes.InternalServerError, entity = "OOPS!")
-        expectRequest().header[`Timeout-Access`].foreach(_.timeoutAccess.updateHandler(_ ⇒ timeoutResponse))
+        expectRequest().header[`Timeout-Access`].foreach(_.timeoutAccess.updateHandler((_: HttpRequest) ⇒ timeoutResponse))
```

Comment:
I suspect this is needed because there are 2 `updateHandler` methods on `akka.http.scaladsl.TimeoutAccess`: one from the javadsl `void updateHandler(Function<HttpRequest, HttpResponse> handler);` and one from the scaladsl `def updateHandler(handler: HttpRequest ⇒ HttpResponse): Unit`. I suspect because the scaladsl HttpResponse also extends the javadsl one, this is now ambiguous?

Reply:
Hmm, I guess that's good enough. We don't have to support source compatibility in all fringe cases.